hooks/useConversations.tsx

```diff
@@ -15,10 +15,15 @@ const useConversations = () => {
   const { address } = useUserProvider();
   const { conversation } = useParams();
   const conversationRef = useRef(conversation as string);
-  const [streamingTitle, setStreamingTitle] = useState("");
+  const [streamingTitle, setStreamingTitle] = useState({
+    title: "",
+    isTikTokAnalysis: false,
+  });
   const [streaming, setStreaming] = useState(false);
   const { selectedArtist } = useArtistProvider();
   const [allConverstaions, setAllConverstaions] = useState<Conversation[]>([]);
+  const pathname = usePathname();
+  const isFunnelPage = pathname.includes("/funnels/tiktok-account-analysis");
```

Comment:
Why does `isFunnelPage` matter?
- actual: tracking `page` to determine conversation
- expected: use `message tools` to determine conversation